Example/VLCUIExample/iOS/OverlayView.swift

```diff
@@ -10,57 +10,87 @@ struct OverlayView: View {
+                    viewModel.proxy.jumpBackward(15)
+                } label: {
+                    Image(systemName: "gobackward.15")
+                        .font(.system(size: 28, weight: .regular, design: .default))
                 }
-            } label: {
-                Group {
+
+                Button {
                     if viewModel.playerState == .playing {
-                        Image(systemName: "pause.circle.fill")
-                    } else if viewModel.playerState == .buffering {
-                        ProgressView()
+                        viewModel.proxy.pause()
                     } else {
-                        Image(systemName: "play.circle.fill")
+                        viewModel.proxy.play()
                     }
+                } label: {
+                    Group {
+                        if viewModel.playerState == .playing {
+                            Image(systemName: "pause.circle.fill")
+                        } else if viewModel.playerState == .buffering {
+                            ProgressView()
+                        } else {
+                            Image(systemName: "play.circle.fill")
+                        }
+                    }
+                    .font(.system(size: 28, weight: .heavy, design: .default))
+                    .frame(maxWidth: 30)
                 }
-                .font(.system(size: 28, weight: .heavy, design: .default))
-                .frame(maxWidth: 30)
-            }
 
-            Button {
-                viewModel.proxy.jumpForward(15)
-            } label: {
-                Image(systemName: "goforward.15")
-                    .font(.system(size: 28, weight: .regular, design: .default))
-            }
+                Button {
+                    viewModel.proxy.jumpForward(15)
+                } label: {
+                    Image(systemName: "goforward.15")
+                        .font(.system(size: 28, weight: .regular, design: .default))
+                }
+                
+                HStack(spacing: 5) {
+                    Text(viewModel.positiveTimeLabel)
+                        .frame(width: 50)
 
-            HStack(spacing: 5) {
-                Text(viewModel.positiveTimeLabel)
-                    .frame(width: 50)
+                    Slider(
+                        value: $currentPosition,
+                        in: 0 ... Float(1.0)
+                    ) { isEditing in
+                        isScrubbing = isEditing
+                    }
 
-                Slider(
-                    value: $currentPosition,
-                    in: 0 ... Float(1.0)
-                ) { isEditing in
-                    isScrubbing = isEditing
+                    Text(viewModel.negativeTimeLabel)
+                        .frame(width: 50)
                 }
-
-                Text(viewModel.negativeTimeLabel)
-                    .frame(width: 50)
-            }
+            }.disabled(viewModel.isRecording)
```

Comment:
Just remove the disabling of these buttons, they can still be used during a recording.